PICS/Resources/Localizable.xcstrings

```diff
@@ -524,7 +524,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "This application demonstrates several Spezi features & modules."
+            "value" : "This application helps to:"
```

Comment:
Please try to come up with a better suited subtitle that fits the content of the rest of the view

Reply:
Tried to use better wording